packages/ra-language-english/index.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
             add: 'Add',
             back: 'Go Back',
             bulk_actions: '1 item selected |||| %{smart_count} items selected',
+            unselect_items: 'Unselect item |||| Unselect %{smart_count} items'
```

Comment:
"Unselect 3 items" is in the tooltip, not as text, so I guess that way it isn't feeling as repetitive. But we can change this text of course. Or just remove the tooltip, since the cross should make sense intuitively. It does to me at least.

Reply:
No please keep it for accessibility

Reply:
> I think it's better to just use 'Unselect' in all cases.

Agreed. 'Unselect' without repeating the number in the Tooltip is fine.

Reply:
btw an `aria-label` would be nice :)